lib/puppet/face/config.rb

```diff
@@ -129,4 +129,45 @@
       nil
     end
   end
+
+  action(:delete) do
+    summary _("Delete a Puppet setting.")
+    #TRANSLATORS '--section' is a command line option and should not be translated
```

Comment:
Was this meant to be above the notes section further down?

Reply:
No, I had added the the option to the argument, but then realized later that it wasn't needed and removed it. 

Removed